lightningd/log.c

```diff
@@ -450,8 +450,10 @@ static int signalfds[2];
 
 static void handle_sighup(int sig)
 {
-	/* This may fail if we're hammered with SIGHUP.  We don't care. */
-	if (write(signalfds[1], "", 1));
+	/* Writes a single 0x00 byte to the signalfds pipe. This may fail if
+	 * we're hammered with SIGHUP.  We don't care. */
+	if (write(signalfds[1], "", 1))
```

Comment:
`clang` will complain if this is not on a separate line to signal explicit opt-out of this case.